src/lib/components/Login.svelte

```diff
@@ -0,0 +1,71 @@
+<script lang="ts">
+	import { isValidEmail } from '$lib/ValidateInputs';
+	import RegisterInput from '../../lib/components/RegisterInput.svelte';
+
+	let email = '';
+	let password = '';
+	let validateIconEmail = '';
+	let validateIconPwd = '';
+	let validateIconEmailColor = '';
+	let validateIconPwdColor = '';
+	let isPasswordEntered: boolean;
+
+	function handleEmailInput(event: Event) {
+		email = (event.target as HTMLInputElement).value;
+	}
+
+	function handlePasswordInput(event: Event) {
+		password = (event.target as HTMLInputElement).value;
+		if (password.length >= 1) {
+			isPasswordEntered = isValidEmail(email);
+		} else {
+			validateIconPwd = '';
+		}
+	}
+
+	$: allInputFieldsFilled = password.length != 0 && email.length != 0 && isPasswordEntered;
+	function handleSubmit() {}
+</script>
+
+<div class="pt-8 rounded-container-token">
+	<div class="justify-center">
+		<div class=" p-2 h4 text-center font-bold">Log In</div>
+		<div class="h-[50vh] flex flex-col">
+			<form class="flex flex-col justify-around items-center w-full" on:submit={handleSubmit}>
+				<div class="flex flex-col w-full pr-3 pl-3 pb-2">
```

Comment:
pr-3 und pl-3 --> da gibts einen command der das in einem macht oder du zentrierst das ganze einfach